kotlin-ring-ui/src/main/kotlin/ringui/Avatar.kt

```diff
@@ -0,0 +1,22 @@
+@file:JsModule("@jetbrains/ring-ui/components/avatar/avatar")
+@file:JsNonModule
+
+package ringui
+
+import react.ComponentClass
+import react.PropsWithClassName
+
+external interface AvatarProps : PropsWithClassName {
+    var dpr: Double
+    var size: Double
+    var style: dynamic
```

Comment:
What is TS `style` type?

Reply:
```js   
 style: PropTypes.object,```

Reply:
`PropsWithStyle`?

Reply:
Then Checkbox component also needs to be changed to ` PropsWithStyle `, as it too uses 
```kotlin
var containerStyle: dynamic
```